Network.cpp

```diff
@@ -957,7 +957,8 @@ void Network::publishHASSConfig(char* deviceType, const char* baseTopic, char* n
                          "",
                          "",
                          mqtt_topic_lock_action,
-                         { { "pl_prs", "lockNgo" }});
+                         { { "enabled_by_default", "true" },
```

Comment:
`enabled_by_default` should be set to `false`

Reply:
Darn your quick ;)
While I'm not awake apparently